### DISCUSSION TITLE
Best way to import a multi-cd "album" which is not on MusicBainz/Discogs

### DISCUSSION THREAD

**Discussion by halloleo** [beetbox/beets#5254] [Q&A]
I have some "albums" which span mutiple CDs. When I rip them on my Mac they come up as `Unknown Artist/Unknown Album/Track 1-01.wav...` for the first CD and as and the second CD `Unknown Artist/Unknown Album/Track 2-01.wav...`

What's the best way to import this with beets and give MusicBainz a chance to find the CDs?

**Comment by DjSlash** [comment]
Many rip software still tries to use CDDB or the likes. That's too bad, because they became obsolete. Anyway, luckily everybody can contribute on the completeness of MusicBrainz (MB). :tada: 

With beets, if the album is on MB, you can provide the release ID during import (choose `enter Id`, copy the URL of the release from MB and paste it).

If the album is **not** on MB, add it on MB before the import.

There are plugins that can help in this process:
- [edit](https://beets.readthedocs.io/en/stable/plugins/edit.html)
- [mbsumbit](https://beets.readthedocs.io/en/stable/plugins/mbsubmit.html)

**Reply by halloleo** [reply]
Thanks for this. 

Though it is still unclear to me how beets can detect _automatically_ what album these `*.wav` files represent.

**Reply by DjSlash** [reply]
It can't. Beets uses existing metadata, but if that isn't present, it will fail.

**Reply by halloleo** [reply]
Interesting, so it does not do this CD checksum thing CDDB did back in the day when it worked?

**Reply by DjSlash** [reply]
No, that's a different kind of process. The rip-software should do that. 

Although, you could use the [chroma](https://beets.readthedocs.io/en/stable/plugins/chroma.html) plugin. It's a bit of a fiddle to set up, but it fingerprints files and that can help with identifying.
